src/pages/tutorials/build-an-nft-marketplace/index.md

```diff
@@ -170,27 +182,10 @@ type storage =
     metadata: NFT.Metadata.t,
     token_metadata: NFT.TokenMetadata.t,
     operators: NFT.Operators.t,
-    token_ids : set<NFT.Storage.token_id>
+    token_ids : set<NFT.token_id>
   };
 
 type ret = [list<operation>, storage];
-
-type parameter =
-  | ["Mint", nat,bytes,bytes,bytes,bytes] //token_id, name , description  ,symbol , ipfsUrl
-  | ["AddAdministrator" , address]
-  | ["Transfer", NFT.transfer]
-  | ["Balance_of", NFT.balance_of]
-  | ["Update_operators", NFT.update_operators];
-
-
-const main = ([p, s]: [parameter,storage]): ret =>
-    match(p, {
-     Mint: (p: [nat,bytes,bytes,bytes,bytes]) => [list([]),s],
-     AddAdministrator : (p : address) => {if(Set.mem(Tezos.get_sender(), s.administrators)){ return [list([]),{...s,administrators:Set.add(p, s.administrators)}]} else {return failwith("1");}} ,
-     Transfer: (p: NFT.transfer) => [list([]),s],
-     Balance_of: (p: NFT.balance_of) => [list([]),s],
-     Update_operators: (p: NFT.update_operator) => [list([]),s],
-     });
 ```
 
 Explanations:
```

Comment:
- Capitalize the first word in a list
- Don't use "we"
- Typo: "the the"
- "storage definition is an extension of the imported library storage, we point to the original types keeping the same naming" -- can you simplify or clarify this?
- What is "tzip-16 compliance"?